src/components/Flipper/Flipper.tsx

```diff
@@ -338,28 +341,31 @@ function Flipper() {
                 <Card.Body>
                     <div id="flipper-card-body">
                         <FlipperFilter onChange={onFilterChange} isLoggedIn={isLoggedIn} isPremium={hasPremium} />
-                        <Form inline >
+                        <hr />
+                        <Form inline style={{ justifyContent: "space-evenly" }}>
                             <Form.Group>
-                                <div>
-                                    <Form.Label htmlFor="autoScrollCheckbox" style={{ float: "left", marginRight: "10px" }}>Auto-Scroll?</Form.Label>
-                                    <Form.Check id="autoScrollCheckbox" checked={autoscroll} onChange={(e) => { _setAutoScroll(e.target.checked) }} type="checkbox" />
+                                <Form.Label htmlFor="autoScrollCheckbox" style={{ marginRight: "10px" }}>Auto-Scroll?</Form.Label>
+                                <Form.Check inline id="autoScrollCheckbox" checked={autoscroll} onChange={(e) => { _setAutoScroll(e.target.checked) }} type="checkbox" />
+                            </Form.Group>
+                            <Form.Group>
+                                <div style={{ display: "contents", cursor: "pointer", marginRight: "10px" }} onClick={clearFlips}>
+                                    <Form.Label>Clear flips!</Form.Label>
+                                    <DeleteIcon color="error" />
                                 </div>
                             </Form.Group>
                             <Form.Group onClick={onArrowRightClick}>
```

Comment:
This isn't the correct click handler for this.
Please display a "Coming Soon" Tooltip for this element and remove the onClick for now